dashboard/test/models/user_helpers_test.rb

```diff
@@ -110,4 +110,18 @@ def create_user_with_username(username)
 
     assert_equal expected_usernames, users.collect(&:username)
   end
+
+  test 'age_range_from_birthday' do
+    assert_equal '18+', UserHelpers.age_range_from_birthday(33.years.ago.to_datetime)
+    assert_equal '18+', UserHelpers.age_range_from_birthday(18.years.ago.to_datetime)
+    assert_equal '13+', UserHelpers.age_range_from_birthday(17.years.ago.to_datetime)
+    assert_equal '13+', UserHelpers.age_range_from_birthday(14.years.ago.to_datetime)
+    assert_equal '13+', UserHelpers.age_range_from_birthday(13.years.ago.to_datetime)
+    assert_equal '8+', UserHelpers.age_range_from_birthday(12.years.ago.to_datetime)
+    assert_equal '8+', UserHelpers.age_range_from_birthday(8.years.ago.to_datetime)
+    assert_equal '4+', UserHelpers.age_range_from_birthday(7.years.ago.to_datetime)
+    assert_equal '4+', UserHelpers.age_range_from_birthday(5.years.ago.to_datetime)
```

Comment:
Weird. I think I've seen this failure before locally too. Might add a quick comment to the file itself?